lib/reducers/create-otp-reducer.js

```diff
@@ -1,34 +1,52 @@
 import clone from 'clone'
 import update from 'immutability-helper'
+import isEqual from 'lodash.isequal'
 
-import queryParams from '../util/query-params'
-import { ensureSingleAccessMode, getUrlParams } from '../util/query'
+import { matchLatLon } from '../util/map'
+import {
+  ensureSingleAccessMode,
+  getDefaultQuery,
+  getTripOptionsFromQuery,
+  getUrlParams
+} from '../util/query'
 import { isTransit, getTransitModes } from '../util/itinerary'
 import { filterProfileOptions } from '../util/profile'
-import { MobileScreens } from '../actions/ui'
+import { getItem, removeItem, storeItem } from '../util/storage'
+import { MainPanelContent, MobileScreens } from '../actions/ui'
 
 const defaultConfig = {
   autoPlan: false,
   debouncePlanTimeMs: 0,
   language: {},
+  operators: [],
   realtimeEffectsDisplayThreshold: 120,
   routingTypes: []
 }
 
-// construct the initial/default query
-const defaultQuery = { routingType: 'ITINERARY' }
-queryParams.filter(qp => 'default' in qp).forEach(qp => {
-  defaultQuery[qp.name] = qp.default
-})
-
+// Load user override settings from local storage.
+// TODO: Make this work with settings fetched from user profile API service.
+const userOverrides = getItem('defaultQuery')
+const defaults = Object.assign(getDefaultQuery(), userOverrides)
+const autoRefresh = getItem('autoRefresh', true)
```

Comment:
It'd be nice to have some comments for what these are all used for. Like for this one, it'd be nice if there was a comment that it's auto refreshing stoptime updates. Maybe this variable could be renamed to be more specific since the vehicle rental layers also auto-refresh.